code/eyetracking/convert.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2024 The Axon Lab <theaxonlab@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# We support and encourage derived works from this project, please read
+# about our expectations at
+#
+#     https://www.nipreps.org/community/licensing/
+#
+from __future__ import annotations
+
+from pathlib import Path
+import re
+import pandas as pd
+
+from eyetrackingrun import EyeTrackingRun, write_bids
+
+
+TASK_TRIGGER_MSG = {
+    "fixation": ("hello", "bye"),
+    "qct": ("hello qct", "bye qct"),
+    "rest": ("start movie", "Bye rs"),
+    "bht": ("hello bht", "Bye bht"),
+}
+
+if __name__ == "__main__":
+    import argparse
+
+    parser = argparse.ArgumentParser(description="Check and fix wrong PA fieldmaps.")
+    parser.add_argument("recordings", type=Path, help="Folder containing EDF files.")
+    parser.add_argument(
+        "bids_file",
```

Comment:
@celprov I've realized that an easy way of organizing the conversion post-hoc (meaning, once the imaging data has been already converted into BIDS) is to give the script the path of the file that this physio or eyetracking (or events file) is associated with.

Reply:
This is to say, I think I will change this in the psychopy2events script.